### ISSUE TITLE
threshold set to -3

### ISSUE THREAD

**Issue by NikitaThomas** [ymli39/DeepSEED-3D-ConvNets-for-Pulmonary-Nodule-Detection#10]
Hi, 

Can you explain why the threshold was set to -3 when calculating the predicted bounding boxes. Does this threshold relate to the probability/confidence score of the pbb. If not, where can I find the cancer probability scores of each nodule detected?

**Comment by ymli39** [comment]
threshold is set to -3 to obtain feature mask further will be used as input for classification network, which in this case we do not have, so please just ignore this threshold.
For probability score, you could find it at saved 'xxx_pbb.npy' file. Please check the evaluation script at LIDC_detector/FROCeval.py.

**Comment by MjdMahasneh** [comment]
@NikitaThomas 

The paper mentions 10-folds validation (which I presume, also means there is 10-fold training), my question is :

1-how is the 10-fold training done using the training script?
2-are these files (luna_train.npy and luna_test.npy) related to the 10-fold training?
3-how are these files generated?

Looking forward to hearing back from you.
